website/content/docs/platform/k8s/vso/index.mdx

```diff
@@ -141,6 +141,8 @@ spec:
 Provides the configuration necessary for the Operator to synchronize a single Vault *PKI* Secret to a single Kubernetes Secret.<br />
 Supported secrets engines: [pki](/vault/docs/secrets/pki)
 
+In this example, the PKI secret engine is mounted at `pki` path. A Vault role named `default` is configured to map to a procedure for generating a certificate.
```

Comment:
Looks good, but I am not sure what is meant by `procedure` here? We may want to link out to the relevant PKI docs for that will go into more details on the how the engine works.

Reply:
We may  want to mention that `path` is under `mount`, and then provide an example of the full/abs path.